### PR TITLE
@task PDA-1351 Add button to test connection with Kafka server in configuration details dialog

### PR DESCRIPTION
**Pull request by dtang-parasoft**

### CODE REVIEW

File: src/main/resources/i18n/aerospace/common.properties [hunk 1]
```diff
@@ -170,7 +170,8 @@ TESTING=Testing...
 VALID_VIRTUALIZE_SERVER_URL=Valid URL.
 INVALID_VIRTUALIZE_SERVER_URL=Invalid URL.
 INVALID_PARASOFT_VIRTUALIZE_SERVER_URL_1=Can't establish connection with Virtualize server.
-INVALID_KAFKA_SERVER_URL=Can't establish connection with kafka server.
+INVALID_KAFKA_SERVER_URL=Can't establish connection with Kafka broker.
+CONNECT_KAFKA_BROKER_SUCCESS=Connect kafka broker successfully.
```
Comment (yshi-parasoft) [inline src/main/resources/i18n/aerospace/common.properties:174]: Please change "kafka" to "Kafka" and change this message to: "Connection to Kafka broker established successfully."

File: src/main/resources/static/common/css/demoAdmin.css [hunk 2]
```diff
@@ -460,6 +460,10 @@ textarea {
     padding-bottom: 7px;
 }
 
+.test_connection_kafka_service {
+    text-align: center;
```
Comment (yshi-parasoft) [inline src/main/resources/static/common/css/demoAdmin.css:464]: Let's add 20px of margin-top.

File: src/main/resources/static/common/js/demoAdmin.js [hunk 3]
```diff
@@ -413,6 +430,9 @@ mod.controller('demo_admin_controller', function($rootScope, $scope, $http, $fil
         }).then(function(result) {
             localStorage.setItem("status", "true");
             localStorage.setItem("save_succeeded", $filter('translate')('SAVING_SUCCEEDS'));
+            if (result.data.data.mqType !== "KAFKA") {
```
Comment (yshi-parasoft) [inline src/main/resources/static/common/js/demoAdmin.js:433]: Please wait for [story_pda1092](https://github.com/parasoft/parasoft-demo-app/pull/210) to merge first. 
This variable name may be changed. 
Also you might have made duplicate changes as the other pull request.

**Comment by yshi-parasoft** [comment]
image
Currently when "Testing..." displays, the UI will move because of it. We should add it to a place that can hold the space in all cases.

File: src/main/resources/static/common/js/demoAdmin.js [hunk 4]
```diff
@@ -291,6 +291,23 @@ mod.controller('demo_admin_controller', function($rootScope, $scope, $http, $fil
         }
     }
 
+    demo.validateKafkaBrokerUrl = function () {
+        demo.disableConnectionKafkaServiceButton = true;
```
Comment (rwang-parasoft) [inline src/main/resources/static/common/js/demoAdmin.js:295]: I think use `demo.isTestingKafkaBrokerUrl` should be better.

File: src/main/resources/templates/demoAdmin.html [hunk 5]
```diff
@@ -263,6 +263,11 @@
                                                     </tr>
                                                     </tbody>
                                                 </table>
+                                                <div class="test_connection_kafka_service" ng-if="options.mqType === 'KAFKA'">
+                                                        <button id="test_connection_kafka_service_button" class="btn btn-default industry_button" translate="TEST_CONNECT"
+                                                                  ng-disabled="demo.disableConnectionKafkaServiceButton" ng-click="demo.validateKafkaBrokerUrl()"></button>
+                                                </div>
+                                                <span ng-show="demo.disableConnectionKafkaServiceButton" class="testing_connection_kafka_service"  translate={{'TESTING'|translate}}></span>
```
Comment (rwang-parasoft) [inline src/main/resources/templates/demoAdmin.html:270]: If we use `ng-show="demo.isTestingKafkaBrokerUrl"` here, it will make sense.

**Pull request merged**